packages/plugin-graphql/src/core/cache.js

```diff
@@ -1,4 +1,4 @@
-import ApolloCore from '@apollo/client/core/core.cjs.js';
+import ApolloCore from '@apollo/client/core/core.cjs';
```

Comment:
I found [this thread](https://github.com/apollographql/apollo-client/issues/10319) and [this thread](https://github.com/apollographql/apollo-client/issues/9925#issuecomment-1191045735) which got me looking at the package contents and and maybe there's an alternative we can't, which happens to be a nice ESM export?  🤩 
https://unpkg.com/browse/@apollo/client@3.7.14/core/index.js

```js
// thinking it would be nice if one of these could work
import { ApolloCore } from '@apollo/client';

// or
import { ApolloCore } from '@apollo/client/core/index';
```

Not sure if it's new or a named or default export but might be worth a try.  I can quickly test that over the weekend as well.